pkg/controller/node/nodecontroller.go

```diff
@@ -720,6 +811,21 @@ func (nc *NodeController) monitorNodeStatus() error {
 	return nil
 }
 
+// We consider the node for scheduling only when its:
+// - NodeReady condition status is ConditionTrue,
+// - NodeOutOfDisk condition status is ConditionFalse,
+// - NodeNetworkUnavailable condition status is ConditionFalse.
+func nodeSchedulable(cond v1.NodeCondition) bool {
```

Comment:
I cannot find anywhere we use it, seems duplicate with unSchedulable fun, or I missed something?